pkg/operator/discover/discover.go

```diff
@@ -207,6 +207,7 @@ func (d *Discover) createDiscoverDaemonSet(namespace, discoverImage, securityAcc
 	} else {
 		logger.Infof("rook-discover daemonset started")
 	}
+	k8sutil.AddRookVersionLabelToDaemonSet(ds)
```

Comment:
Discover daemon is not a CSI resource. This should be a change in a separate PR.